pkg/services/sqlstore/db_hook.go

```diff
@@ -0,0 +1,69 @@
+package sqlstore
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"xorm.io/core"
+)
+
+var (
+	databaseQueryCounter  *prometheus.CounterVec
+	databaseQueryDuration *prometheus.SummaryVec
+)
+
+func init() {
+	databaseQueryCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "grafana",
+		Name:      "database_queries_total",
+		Help:      "The total amount of Database queries",
+	}, []string{"status"})
+
+	databaseQueryDuration = prometheus.NewSummaryVec(prometheus.SummaryOpts{
+		Namespace:  "grafana",
+		Name:       "database_queries_duration",
+		Help:       "Database query duration",
+		Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
+	}, []string{"status"})
```

Comment:
so nice! :heart:

Reply:
Yea I think this will be good once it's been refactored a bit :)